assets/main.scss

```diff
@@ -228,6 +228,8 @@ main article img{
   display: flex;
   border: 5px solid white;
   box-shadow: 0 1px 3px black;
+  background: white;
```

Comment:
I can't see any harm in forcing it. I don't think images should be responsible for their own presentation in a page.